test/images/iperf/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=gcr.io/google_containers/ubuntu-slim:0.12
```

Comment:
@luxas @ixdy I can work on this to change to debian-base image, but can someone tell are the extra benefit I'll get by doing that?